dojo/filters.py

```diff
@@ -228,18 +228,6 @@ def __init__(self, *args, **kwargs):
             (key, value[0]) for key, value in six.iteritems(self.options)]
         super(MetricsDateRangeFilter, self).__init__(*args, **kwargs)
 
-        try:
-            earliest_finding = Finding.objects.earliest('date')
-        except Finding.DoesNotExist:
-            earliest_finding = None
-
-        if earliest_finding:
-            start_date = local_tz.localize(datetime.combine(
-                earliest_finding.date, datetime.min.time())
-            )
-            self.start_date = _truncate(start_date - timedelta(days=1))
-            self.end_date = _truncate(now() + timedelta(days=1))
-
```

Comment:
We cannot make DB queries in the __init__ method. Not sure why previous versions of Django did not complain about it.